src/crypto/tls/common.go

```diff
@@ -190,6 +191,23 @@ var supportedSignatureAlgorithms = []SignatureScheme{
 	ECDSAWithSHA1,
 }
 
+// supportedSignatureAlgorithmsDC contains the signature and hash algorithms that
+// the code advertises as supported in a TLS 1.2+ ClientHello and in a TLS 1.2+
+// CertificateRequest. The two fields are merged to match with TLS 1.3.
+// Note that in TLS 1.2, the ECDSA algorithms are not constrained to P-256, etc.
+// This excludes 'rsa_pss_rsae_' algorithms.
```

Comment:
DC aren't supported in TLS 1.2?

Reply:
No, they are not. It got bumped in draft 07 I think..

Reply:
Okay, then this should probably be reworded.

Reply:
Oh, right. This was a comment I copied.